meinberlin/apps/budgeting/api.py

```diff
@@ -8,11 +9,17 @@
 from .serializers import ProposalSerializer
 
 
+class BudgetPagination(PageNumberPagination):
+    page_size_query_param = 'page_size'
```

Comment:
we already talked about it, but I like to set the pagination size here. :)
just `page_size = 15` should work
The current one is set in the AbstractIdeaListView and is 15 https://github.com/liqd/a4-meinberlin/blob/73a9d01832939580d3574ca037df1327eacba235/meinberlin/apps/ideas/views.py#L60
Maybe we can add a comment here to add a more general IdeaPagination once we use this for more Idea lists?

Reply:
alright, will make it settable in backend and set it to 15.
shall i add the comment to `budgeting/api.py` or rather to /ideas/views.py`?

Reply:
I would add the comment directly here to beudgeting/api. I hope that we check and read there when we want to use React for another idea list.